src/catalog/postgres/pg_core_impl.cpp

```diff
@@ -7,6 +7,7 @@
 #include "catalog/postgres/pg_class.h"
 #include "catalog/postgres/pg_index.h"
 #include "catalog/postgres/pg_namespace.h"
+#include "catalog/postgres/pg_statistic.h"
```

Comment:
Could you move all the changes from this file out of `pg_core_impl` and into its own `pg_statistic_impl`?

The idea is that `pg_core` is the absolute minimum necessary to spin up the rest of the system.
We spin up the system now just fine without stats, so it doesn't belong in `core`.

Reply:
Done

Reply:
thanks!